de-lib/src/main/java/org/iplantc/de/client/services/DiskResourceServiceFacade.java

```diff
@@ -36,7 +38,7 @@
      * @param folder The parent of the subfolders to refresh from the service.
      * @param callback executed when RPC call completes.
      */
-    void refreshFolder(Folder folder, final AsyncCallback<List<Folder>> callback);
+    void refreshFolder(Folder folder, final DECallback<List<Folder>> callback);
```

Comment:
Why is it these ones are `DECallback` rather than `DataCallback`? Couldn't quite tell what the difference was supposed to be (mostly looking through the PR to see if there's any tricky multi-service-endpoint edge cases I could rustle up)